APIReplayDownloader/database_lb_test.py

```diff
@@ -0,0 +1,10 @@
+import database
+
+lb_db = database.open_db("db/leaderboard.fs")
```

Comment:
Try to avoid abbreviations like "lb". Things like "db" are more acceptable since its so universally known.